lib/main.js

```diff
@@ -95,15 +97,15 @@ const Hydrogen = {
         const editorSubscriptions = new CompositeDisposable();
         editorSubscriptions.add(
           editor.onDidChangeGrammar(() => {
-            store.setGrammar(editor);
+            store.updateEditorAndGrammar(editor);
           })
         );
 
         if (isMultilanguageGrammar(editor.getGrammar())) {
           editorSubscriptions.add(
             editor.onDidChangeCursorPosition(
               _.debounce(() => {
-                store.setGrammar(editor);
+                store.updateEditorAndGrammar(editor);
```

Comment:
It was a left-over from a refactor I tried. I've just pushed a commit to restore `setGrammar`.